dockerfiles/Dockerfile-toolchain

```diff
@@ -139,5 +141,17 @@ RUN curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash - \
 # easier.
 # RUN sudo ln -sf /usr/share/zoneinfo/UTC /etc/localtime
 
+# --- Build and install capnproto go compiler
```

Comment:
This is effectively a build dependency and it should be handled as part of the build-deps stage of the rosetta dockerfile. For this toolchain container just update the sha hash in the `FROM` directive.